leapp/utils/output.py

```diff
@@ -28,9 +28,23 @@ def report_errors(errors):
 ============================================================{reset}\n\n""".format(red=red, reset=reset))
 
 
+def report_report(path):
```

Comment:
Can you please rename the function to something more self-speaking?
report_info?

Reply:
Changed to report_info.